code/modules/research/designs/limbgrower_designs.dm

```diff
@@ -279,3 +312,13 @@
 	desc = "Contains designs for ethereal organs for the limbgrower - Ethereal tongue and stomach."
 	id = "limbdesign_ethereal"
 	build_path = /obj/item/disk/design_disk/limbs/ethereal
+
+/obj/item/disk/design_disk/limbs/hemophage
```

Comment:
I feel as if this is placed weirdly. Could you break all the edits you made to this file and put them into a new file in the modular_zubbers folder? This way it will have permanence.